grunt.js

```diff
@@ -314,6 +300,23 @@ module.exports = function(grunt) {
         src: ['build/temp/video.js', 'node_modules/vtt.js/dist/vtt.js'],
         dest: 'build/temp/video.js',
       },
+    },
+    concurrent: {
+      options: {
+        logConcurrentOutput: true
+      },
+      // Run multiple watch tasks in parallel
+      // Needed so watchify can cache intelligently
+      watchAll: [
+        'watch',
+        'browserify:watch',
```

Comment:
The order doesn't seem to have a significant impact on when they're run (or rather when they finish), from what I've seen. But I'll try it. I think we still want `watch` first though, because if you're developing the skin you don't want to wait for the rest, and also the tests depend on the skin being available.